crates/optimism/rpc/src/eth/ext.rs

```diff
@@ -19,33 +20,48 @@ const MAX_CONDITIONAL_EXECUTION_COST: u64 = 5000;
 /// Separate from [`super::OpEthApi`] to allow to enable it conditionally,
 #[derive(Clone)]
 #[allow(dead_code)]
-pub(crate) struct OpEthApiExt<N: OpNodeCore> {
-    /// Gateway to node's core components.
-    inner: Arc<OpEthApiInner<N>>,
+#[derive(Debug)]
+pub struct OpEthExtApi<N: OpNodeCore> {
```

Comment:
I think we can now introduce Pool,Provider generics, then this isn't tied to the N:OpNodeCore

Reply:
good point, done